examples/events.py

```diff
@@ -0,0 +1,123 @@
+"""
+Example showing events and clicking on meshes.
+
+Clicking on a cube will select it. Hovering the mouse over a cube
+will show a hover color. Double-clicking a cube will select all
+the items from that group (because the group has a double-click
+event handler).
+"""
+
+from random import randint, random
+
+from wgpu.gui.auto import WgpuCanvas, run
+import pygfx as gfx
+
+
+canvas = WgpuCanvas()
+renderer = gfx.renderers.WgpuRenderer(canvas)
+
+camera = gfx.PerspectiveCamera(70, 16 / 9)
+camera.position.z = 400
+
+controls = gfx.OrbitControls(camera.position.clone())
+controls.add_default_event_handlers(renderer, canvas, camera)
+
+scene = gfx.Scene()
+
+geometry = gfx.box_geometry(40, 40, 40)
+material = gfx.MeshPhongMaterial()
+selected_material = gfx.MeshPhongMaterial(color="#FF0000")
+hovered_material = gfx.MeshPhongMaterial(color="#FFAA00")
+
+state = {
+    "selected": None,
+    "hovered": None,
+}
```

Comment:
I think you can make the example more readable if you just have two variables `selected_obj` and `hovered_obj`